packages/neuron-wallet/src/utils/ckb-rpc.ts

```diff
@@ -11,11 +11,11 @@ import {
   PayloadInBatchException,
   IdNotMatchedInBatchException,
 } from '@ckb-lumos/rpc/lib/exceptions'
-import https from 'https'
-import http from 'http'
 import { request } from 'undici'
 import CommonUtils from './common'
 import { NetworkType } from '../models/network'
+import { RPCConfig } from '@ckb-lumos/rpc/lib/types/common'
+import { CKBComponents } from '@ckb-lumos/rpc/lib/types/api'
```

Comment:
These two imports should be marked as `import type` and moved to the top

Reply:
switched to `import type`